joomla/joomla-cms
# [4.0] Group contact toolbar buttons

### Summary of Changes
This PR continues the grouping of action buttons in the backend (see https://github.com/joomla/joomla-cms/pull/22832, https://github.com/joomla/joomla-cms/pull/22393, https://github.com/joomla/joomla-cms/pull/23337 and https://github.com/joomla/joomla-cms/pull/23336). This time the contact buttons.


### Testing Instructions
Go to the contact manager in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a contact item.
